src/qute/languageServer/client.ts

```diff
@@ -71,59 +70,50 @@ export function connectToQuteLS(context: ExtensionContext, api: JavaExtensionAPI
 
     const serverOptions = prepareExecutable(requirements);
     const quteLanguageClient = new LanguageClient('qute', 'Qute Support', serverOptions, clientOptions);
-    context.subscriptions.push(quteLanguageClient.start());
-
-    return quteLanguageClient.onReady().then(async () => {
-      bindQuteRequest('qute/template/project');
-      bindQuteRequest('qute/template/projectDataModel');
-      bindQuteRequest('qute/template/userTags');
-      bindQuteRequest('qute/template/javaTypes');
-      bindQuteRequest('qute/template/resolvedJavaType');
-      bindQuteRequest('qute/template/javaDefinition');
-      bindQuteRequest('qute/template/javadoc');
-      bindQuteRequest('qute/template/generateMissingJavaMember');
-      bindQuteRequest('qute/java/codeLens');
-      bindQuteRequest('qute/java/diagnostics');
-      bindQuteRequest('qute/java/documentLink');
-      bindQuteNotification('qute/dataModelChanged');
-
-      registerQuteExecuteWorkspaceCommand(context, quteLanguageClient);
-      // Refresh the Qute context when editor tab has the focus
-      context.subscriptions.push(
-        window.onDidChangeActiveTextEditor(async editor => {
-          await synchronizeQuteValidationButton(editor);
-        })
-      );
-      // Refresh the Qute context when the language id changed (HTML -> Qute HTML or Qute HTML -> HTML)
-      context.subscriptions.push(
-        workspace.onDidOpenTextDocument(async (document) => {
-          // when settings.json is updated, onDidOpenTextDocument is called,
-          // the Qute context must be refreshed only for the activate text editor.
-          if (window.activeTextEditor?.document === document) {
-            await synchronizeQuteValidationButton(window.activeTextEditor);
-          }
-          // Display the experimental Qute validation pop-up if it hasn't been displayed and a Qute file is open
-          if (!hasShownQuteValidationPopUp(context) && document.languageId.includes('qute')) {
-            showQuteValidationPopUp(context);
-          }
-        })
-      );
-      // Display the experimental Qute validation pop-up if it hasn't been displayed and a Qute file is open
-      if (!hasShownQuteValidationPopUp(context)) {
-        for (const textDocument of workspace.textDocuments) {
-          if (textDocument.languageId.includes('qute')) {
-            showQuteValidationPopUp(context);
-          }
+    await quteLanguageClient.start();
+
+    bindQuteRequest('qute/template/project');
```

Comment:
These requests have been removed from the list, and should be added back
- `'qute/template/javadoc'`
- `'qute/template/generateMissingJavaMember'`

Reply:
Ouch my bad -( Fixed.